backend/api/models.py

```
@@ -173,4 +179,53 @@ class Meta:
+    )
+    
+    form = models.ForeignKey(EvaluationForm, on_delete=models.CASCADE, related_name='questions')
+    question_text = models.CharField(max_length=500)
+    question_type = models.CharField(max_length=20, choices=QUESTION_TYPES, default='rating')
+    options = models.JSONField(null=True, blank=True)  # For multiple choice questions
+    required = models.BooleanField(default=True)
+    order = models.PositiveIntegerField(default=0)
+    
+    class Meta:
+        db_table = 'evaluation_questions'
+        ordering = ['order']
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)